lib/bootstrap.js

```diff
@@ -45,13 +47,39 @@ async function runCLI(cli, commandIsUsed) {
     let args;
     const helpFlagExists = isFlagPresent(process.argv, 'help');
     const versionFlagExists = isFlagPresent(process.argv, 'version');
+    const nodeArgsExists = isFlagPresent(process.argv, 'node-args');
 
     if (helpFlagExists) {
         cli.runHelp(process.argv);
         return;
     } else if (versionFlagExists) {
         cli.runVersion();
         return;
+    } else if (nodeArgsExists) {
+        args = cmdArgs(core, { stopAtFirstUnknown: false, partial: true });
+        const [, , ...rawArgs] = process.argv;
+        const cliArgs = [];
+        const nodeArgs = [];
+        let isNodsArg = false;
```

Comment:
Would you mind extract this logic inside a function and add some tests? 

```js
function parseArgs(rawArgs) {
  // ..
   return {
       cliArgs,
       nodeArgs
   }
}
```

Reply:
I’ll deal with it ☺️